bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/base/SEDFactory.java

```diff
@@ -338,5 +338,13 @@ public SourceResult(final VisitableSampledSpectrum sed, final VisitableSampledSp
         }
     }
 
+    private static String airmassCategory(final double airmass) {
```

Comment:
Probably yes. Although here it really is only about creating a string representation, I know that this concept of an airmass category is used in at least one more place in the ITC. But let me tackle that in a separate PR.